lib/grpnames.gd

```diff
@@ -24,8 +24,8 @@
 ##
 ##  <Description>
 ##    For normal subgroups <A>U</A> and <A>V</A> of <A>G</A>,
-##    IsTrivialNormalIntersection returns
-##    true if <A>U</A> and <A>V</A> intersect trivially, and false otherwise.
+##    <Ref Oper="IsTrivialNormalIntersection"/> returns <K>true</K> if
```

Comment:
Hmm, I am sorry for the conradictory message, but this actually seems a bit odd -- it's kind of recursive: a function's description linking to itself... :-). Do we do this elsewhere? I'd hope we do not... How about instead rephrasing this as "This operation returns ..." ? Or keep the name, but in a <K> or <C> tag... ? I wonder if there are other examples of functions/operations referencing themselves in the manual, and how they solved this?